music21/romanText/writeRoman.py

```diff
@@ -0,0 +1,348 @@
+
+        self.combinedList = [f'Composer: {self.composer}',
+                             f'Title: {self.title}',
+                             f'Analyst: {self.analyst}',
+                             f'Proofreader: {self.proofreader}']
+        if self.note:  # optional
+            self.combinedList.append(f'Note: {self.note}')
+
+        self.currentKey = ''
+
+    def _prepTitle(self):
+        '''
+        In the absence of a user-defined title,
+        attempt to prepare one from the score metadata looking at each of
+        the title, movementNumber and movementName attributes.
+        Failing that, a placeholder 'Unknown' stands in.
+        '''
+
+        if not self.score.metadata:
+            return
+
+        workingTitle = []
+
+        if self.score.metadata.title:
+            workingTitle.append(self.score.metadata.title)
+        if self.score.metadata.movementNumber:
+            workingTitle.append(f'- No.{self.score.metadata.movementNumber}:')  # Spaces later
+        if self.score.metadata.movementName:
+            if self.score.metadata.movementName != self.score.metadata.title:
+                workingTitle.append(self.score.metadata.movementName)
+
+        if len(workingTitle) > 0:
+            self.title = ' '.join(workingTitle)
+
+# ------------------------------------------------------------------------------
+
+    def prepList(self):
+        '''
+        Prepares a sequential list of text lines, with timeSignatures and romanNumerals
+        adding this to the (already prepared) metadata preamble ready for printing.
+        '''
+
+        for m in self.measureNumbers:
+
+            thisMeasure = self.container.measure(m)
+
+            # TimeSignatures
+            tsThisMeasure = [t for t in thisMeasure.getElementsByClass('TimeSignature')]
+            if tsThisMeasure:
+                firstTS = tsThisMeasure[0]
+                self.combinedList.append(f'\nTime Signature: {firstTS.ratioString}')
+                if len(tsThisMeasure) > 1:
+                    msg = 'further time signature change(s) unprocessed: ' \
+                          f'{[x.ratioString for x in tsThisMeasure[1:]]}'
+                    self.combinedList.append(f'Note: {msg}')
+
+            # RomanNumerals
+            measureString = ''  # Clear for each measure
+            rnsThisMeasure = [r for r in thisMeasure.getElementsByClass('RomanNumeral')]
+            if rnsThisMeasure:  # probably redundant
+                for rn in rnsThisMeasure:
+                    chordString = self.getChordString(rn)
+                    measureString = rnString(measure=m,
+                                             beat=rn.beat,
+                                             chordString=chordString,
+                                             inString=measureString,  # Creating update
+                                             )
+
+            self.combinedList.append(measureString)
```

Comment:
maybe
```python
if measureString:
    self.combinedList.append(measureString)
```
I noticed that when there is no RNA annotation for a specific measure, a blank line is generated, but that doesn't seem to be the convention at When-in-Rome/Corpus

Reply:
Ah thanks! Missed this before.